function.php

```diff
@@ -1,24 +1,107 @@
 <?php
 require_once 'db.php';
 
-function getData($db) {
+/**
+ * getData function gets data from the DB
+ * @param $db
+ * @return mixed
```

Comment:
docbloc says mixed when it is type hinted to array

Reply:
ok fixed